src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25473.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if !MACCATALYST
+using Microsoft.Maui.Platform;
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25473 : _IssuesUITest
+	{
+		public Issue25473(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "MAUI Entry in Windows always shows ClearButton despite ClearButtonVisibility set to 'Never'";
+
+		[Test]
+		[Category(UITestCategories.Entry)]
+		public void VerifyEntryClearButtonVisibilitySetToNever()
+		{
+			App.WaitForElement("EntryField");
+			App.Tap("EntryField");
+			App.EnterText("EntryField", "Entry");
+			VerifyScreenshot();
```

Comment:
The test is failing on Android and iOS. There are slightly differences in the snapshot, mostly related with the keyboard.
![image](https://github.com/user-attachments/assets/283c5f1a-d984-434b-8acd-3072ad4a4fcb)
Recommendation, close the keyboard before verify the screenshot or unfocus the Entry. The cursor animation or the keyboard could impact in the screenshot validation.

Reply:
There are a couple of test failing on iOS:
![image](https://github.com/user-attachments/assets/2cec4d3a-efcd-4487-a6a2-1805e452d1ed)

The visual differences are related with the keyboard localization and with the cursor position. 
Need to close the keyboard  `App.DismissKeyboard();` and move the focus.

Reply:
@jsuarezruiz To accurately ensure ClearButtonVisibility, the entry must stay focused. Since the entry with focused state causes the keyboards to remain open in the view, resulting in failures, I changed the test case to exclude other platforms because the actual fix only relates to Windows.